service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml

```diff
@@ -0,0 +1,1417 @@
+databaseChangeLog:
+  - changeSet:
+      id: "update assemble_refbased inputs/outputs to match the target workspace for public preview"
+      author: mspector
+      changes:
+        - update:
+            tableName: run_set
+            where: run_set_id='10000000-0000-0000-0000-000000000005'
+            columns:
+              - column:
+                  name: input_definition
+                  value: |
+                    [
+                      {
+                        "input_name": "assemble_refbased.reads_unmapped_bams",
+                        "input_type": {
+                          "type": "array",
+                          "array_type": {
+                            "type": "primitive",
+                            "primitive_type": "File"
+                          },
+                          "non_empty": true
```

Comment:
Sorry if this is a naive question but does this basically say that this is not an optional Array input?

Reply:
Non empty means it's an error if the array is empty (eg the difference between `null` vs `[]`)

Reply:
Ah I see 👍